docs/source/developer_docs.rst

```diff
@@ -1,7 +1,22 @@
 Developer Docs
 ===============
 
-The Jupyter widgets packages are developed in the `https://github.com/jupyter-widgets/ipywidgets <https://github.com/jupyter-widgets/ipywidgets>`_ git repository.
+The Jupyter widgets packages are developed in the `https://github.com/jupyter-widgets/ipywidgets <https://github.com/jupyter-widgets/ipywidgets>`_ git repository. See the issue tracker, README, and other Github documents for the most recent information.
+
+
+Scope of Ipywidgets
+-------------------
+
+``ipywidgets`` is a framework to provide eventful python objects that have a representation in the browser (see :doc:`examples/Widget Basics.ipynb`for more on the definition of widgets). This requires two components:
+
+1. The framework for widget interactions between the widgets represented in the Python kernel and the (javascript) representation of the widgets in the browser.
+2. A set of implemented widgets that *use* this framework.
```

Comment:
```suggestion
2. A basic, lightweight set of form controls that *use* this framework, based on standard HTML form controls. These included controls include a text area, text box, select and multiselect controls, checkbox, etc., plus a few more advanced controls that are very popular, such as a slider and basic tab panels.
```

Reply:
(I have some minor quibbles there which I'll put in a follow-on after commiting these suggestions)